javascript.md

```diff
@@ -1379,8 +80,6 @@ mainmenu: true
   - [innerHTML vs textContent for script text](http://jsperf.com/innerhtml-vs-textcontent-for-script-text)
   - [Long String Concatenation](http://jsperf.com/ya-string-concat)
 
-
-
 ## Resources
```

Comment:
I think both can be removed.  They are in the airbnb guide
https://github.com/airbnb/javascript#performance